src/_includes/_layouts/Mission.jsx

```diff
@@ -30,12 +30,12 @@ export default ({ comp, values }) => (
               text={"Read More"}
             />
           </div>
-          <div className="lg:w-5/12 lg:absolute lg:right-20 lg:full flex items-end max-lg:flex-1 max-lg:px-10">
-            <div className="max-lg:p-8">
+          <div className="lg:w-5/12 lg:absolute lg:right-20 lg:top-1/2 lg:-translate-y-1/2 flex items-center justify-center max-lg:mt-8 max-lg:px-10">
```

Comment:
Just wanted to share here that a slightly better (more agnostic) approach for the CSS styling would be to use flexbox rather that position absolute to place the image on the right side of the div. 

_**Note**: Absolute positioning an element is good when you want to overlay a div upon another but we can actually take advantage of having a flex display here since it is side-by-side_ 🥳 

Using flexbox allows you to have less CSS classes and prevent something like "image growing larger than the container div" issues from happening as flex parents can contains it's own children pretty easily. Another reason is that hard coding, pixel values is typically frowned upon [[ref](https://github.com/andredesousa/css-best-practices?tab=readme-ov-file#avoid-hard-coded-values)]. So whenever you have to hardcode pixels it usually means there is a cleaner approach somewhere.